planning/motion_velocity_planner/autoware_motion_velocity_planner_common/include/autoware/motion_velocity_planner_common/planner_data.hpp

```diff
@@ -21,6 +21,7 @@
 #include <autoware/route_handler/route_handler.hpp>
 #include <autoware/velocity_smoother/smoother/smoother_base.hpp>
 #include <autoware_utils/geometry/boost_polygon_utils.hpp>
+#include <autoware_utils/ros/parameter.hpp>
```

Comment:
For any new code you write, please make sure to use at least the split `autoware_utils` package.

```suggestion
#include <autoware_utils_rclcpp/parameter.hpp>
```